packages/mds-ingest-service/service/validators.ts

```diff
@@ -201,3 +202,32 @@ export const { validate: validateUUIDs } = SchemaValidator<UUID[]>({
   type: 'array',
   items: uuidSchema
 })
+
+export const { validate: validateEventAnnotationDomainCreateModels } = SchemaValidator<
+  EventAnnotationDomainCreateModel[]
+>({
+  type: 'array',
+  items: {
+    type: 'object',
+    properties: {
+      device_id: { type: 'string', format: 'uuid' },
+      timestamp: { type: 'integer' },
```

Comment:
In other places we use:
```ts
// Timestamp Schema that ensures milliseconds
const TimestampSchema = (options = {}) => ({
  type: 'integer',
  minimum: 100_000_000_000,
  maximum: 99_999_999_999_999,
  ...options
})
```

We really should have a library of these sub schemas.